tools/dependency/release_dates.py

```diff
@@ -51,6 +60,59 @@ def verify_and_print_latest_release(dep, repo, metadata_version, release_date):
         print(
             f'{Fore.YELLOW}*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}>: '
             f'{latest_release.tag_name}@<{latest_release.created_at}>{Style.RESET_ALL}')
+        # check for --check_deps flag, To run this only on github action schedule
+        # and it does not bloat CI on every push
+        if create_issue:
+            create_issues(dep, metadata_version, release_date, latest_release)
+
+
+# create issue for stale dependency
+def create_issues(dep, metadata_version, release_date, latest_release):
+    """Create issues in GitHub.
+
+    Args:
+        dep : name of the deps
+        metadata_version :
+        release_date : old release_date
+        latest_release : latest_release (name and date )
+    """
+    access_token = os.getenv('GITHUB_TOKEN')
+    git = github.Github(access_token)
+    repo = git.get_repo('envoyproxy/envoy')
+    # Find GitHub label objects for LABELS.
+    labels = []
+    for label in repo.get_labels():
+        if label.name in LABELS:
+            labels.append(label.name)
+    if len(labels) != len(LABELS):
+        raise DependencyUpdateError('Unknown labels (expected %s, got %s)' % (LABELS, labels))
+    body = f'*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}>:{latest_release.tag_name}@<{latest_release.created_at}>'
+    title = f'Newer release available {dep}: {latest_release.tag_name}'
+    if issues_exist(title, git):
+        print("Issue with %s already exists" % title)
+        print('  >> Issue already exists, not posting!')
+        return
+    else:
+        print('Creating issues...')
+        try:
+            repo.create_issue(title, body=body, labels=LABELS)
+        except github.GithubException as e:
+            print(f'UNABLE to create issue, received error: {e}. Add them to the Envoy proxy org')
```

Comment:
also, by capitalize i was meaning just the first letter - this should be `Unable`

Reply:
ohkay gotcha !